diagnostic.md

```diff
@@ -21,28 +21,47 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET 'http://www.example.com/donuts/'
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+//I think you still need to get the ID's of every indiviudal donut if you want a list of all the donuts...
+
+let getDonuts = function () {
+  return $.ajax({
+    url: http://www.example.com/donuts,
+    method: 'GET',
+  });
+};
 ```
 
 ## Request a Single Resource
 
 Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
+//
+
 ```sh
-# your answer here
+//Not sure about this curl request... Does the URL know what we mean by // //singleDonut yet?
+
+curl --include --request GET 'http://www.example.com/donuts/singleDonut/'
```

Comment:
It doesn't.  You need to provide an id to the URL.  e.g., `curl --include --request GET http://www.example.com/donuts/1`.